common/predictive-text/build.sh

```diff
@@ -1,53 +1,62 @@
-#! /bin/bash
+#!/bin/bash
 #
 # Compiles the Language Modeling Layer for common use in predictive text and autocorrective applications.
 # Designed for optimal compatibility with the Keyman Suite.
 #
 
-display_usage ( ) {
-  echo "build.sh [-clean]"
-  echo
-  echo "  -clean              to erase pre-existing build products before a re-build"
+EMBEDDED_WORKER=embedded_worker.js
+
+# Build the worker and the main script.
+build ( ) {
+  # Build worker first; the main file depends on it.
+  # Then wrap the worker; Then build the main file.
+  npm run build-worker && wrap_worker && npm run build-main
```

Comment:
I'm not a fan of off-loading this stuff into `npm run` commands - while it's technically fine now, there's a chance that as development proceeds, we'll need additional steps in one or more of the build processes.

I'll admit that it is nice in how it conditionally chains the commands, dependent upon success... though I do worry a bit about the inability to give distinct error messages at each step.  Granted, "a bit."